### ISSUE TITLE
Why is body-color statically mapped to gray-base?

### ISSUE THREAD

**Issue by LeeiFrankJaw** [coreui/coreui#131]
https://github.com/coreui/coreui/blob/b5aa85f1b09a28cdcbeded3da996699373424cc3/scss/variables/_body.scss#L14

Can anyone knowledgeable about this library kindly tell me why this is the case?

I tried to update `$body-color` without seeing any effect and finally find out this.  Is this really by design or by mistake?

The best way to walk around this for me is to modify the `$body-theme-map` as per [documentation][1].

Intuitively, I wrote the following  in my custom style file.

```scss
$body-theme-map: (
  "body-color": $body-color
);
```

But the above won't work since `$body-theme-map` has a nested structure.  Then I tried as follows.

```scss
$body-theme-map: (
  default: ("body-color": $body-color)
);
```

However, that does not work neither since `map-merge` does not merge maps recursively.  And finally, this is what works for me.

```scss
$body-theme-map: (
  default: (
    "body-bg":     $body-bg,
    "body-color": $body-color
  )
);
```

It is lucky that `$body-theme-map` is such a small map. If it is big, I have to copy all the definition from the source code.

 [1]: https://coreui.io/docs/getting-started/theming/#modify-map

**Comment by mrholek** [comment]
@LeeiFrankJaw it is a bug, we will fix it next release.

**Comment by mrholek** [comment]
Fixed in v3.3.0
